tests/appsec/waf/test_addresses.py

```diff
@@ -45,8 +45,8 @@ def test_query_encoded(self):
 
     def test_query_with_strict_regex(self):
         """ AppSec catches attacks in URL query value, even with regex containing"""
-        r = self.weblog_get("/waf/", params={"value": "0000012345"})
-        interfaces.library.assert_waf_attack(r, pattern="0000012345", address="server.request.query")
+        r = self.weblog_get("/waf/", params={"value": "sleep(1)"})
+        interfaces.library.assert_waf_attack(r, pattern="sleep(1)", address="server.request.query")
```

Comment:
are they really equivalent?
I wonder what "_with_strict_regex" really is about 🤔